src/thunderbots/software/test/primitive/primitive.cpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * This file contains the unit tests for the Primitive class
+ */
+
+#include "ai/primitive/primitive.h"
+
+#include <gtest/gtest.h>
+
+#include "ai/primitive/move_primitive.h"
+
+TEST(PrimitiveTest, create_message_empty_test)
+{
+    MovePrimitive move_prim = MovePrimitive(int(), Point(), Angle(), double());
```

Comment:
Since you're comparing int() to int() later on, please use an actual declared number.

Reply:
Removed all int() and double()